src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/49.diff.sql

```diff
@@ -0,0 +1,587 @@
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_ReferenceSearchParam_IsHistory')
+ALTER TABLE dbo.ReferenceSearchParam ADD CONSTRAINT DF_ReferenceSearchParam_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_QuantitySearchParam_IsHistory')
+ALTER TABLE dbo.QuantitySearchParam ADD CONSTRAINT DF_QuantitySearchParam_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_NumberSearchParam_IsHistory')
+ALTER TABLE dbo.NumberSearchParam ADD CONSTRAINT DF_NumberSearchParam_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_DateTimeSearchParam_IsHistory')
+ALTER TABLE dbo.DateTimeSearchParam ADD CONSTRAINT DF_DateTimeSearchParam_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_CompartmentAssignment_IsHistory')
+ALTER TABLE dbo.CompartmentAssignment ADD CONSTRAINT DF_CompartmentAssignment_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.default_constraints WHERE name = 'DF_UriSearchParam_IsHistory')
+ALTER TABLE dbo.UriSearchParam ADD CONSTRAINT DF_UriSearchParam_IsHistory DEFAULT 0 FOR IsHistory
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'UriSearchParamList')
+CREATE TYPE dbo.UriSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,Uri                      varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+
+   PRIMARY KEY (ResourceTypeId, ResourceSurrogateId, SearchParamId, Uri)
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenTokenCompositeSearchParamList')
+CREATE TYPE dbo.TokenTokenCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId            smallint NOT NULL
+   ,ResourceSurrogateId       bigint   NOT NULL
+   ,SearchParamId             smallint NOT NULL
+   ,SystemId1                 int      NULL
+   ,Code1                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,SystemId2                 int      NULL
+   ,Code2                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow2             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenTextList')
+CREATE TYPE dbo.TokenTextList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,Text                     nvarchar(400) COLLATE Latin1_General_CI_AI NOT NULL
+
+   --PRIMARY KEY (ResourceTypeId, ResourceSurrogateId, SearchParamId, Text)
```

Comment:
Should we leave this statement as a comment?

Reply:
Added comment in main code.